src/quote/swap/adapters/zeroex_v2/utils.ts

```diff
@@ -31,7 +32,10 @@ export const exchangeFrom0xSource: { [key: string]: Exchange } = {
 // docs: https://0x.org/docs/api#tag/Sources
 export function getExcludedSources(chainId: number) {
   switch (chainId) {
-    // For now, no need to exclude any sources.
+    case arbitrum.id:
+      return 'WOOFi_V2'
+    case base.id:
+      return 'WOOFi_V2'
```

Comment:
Woofi casues issues on Arbitrum and Base - as seen from Ed's feedback and also testing locally.